packages/pagination/src/Page.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button } from '../../button/src/index.js';
+import { classNames } from "@chbphone55/classnames";
+import { i18n } from '@lingui/core';
+import { pagination as ccPagination } from '@warp-ds/css/component-classes';
+import React from 'react';
+
+export type PageProps = {
+  /**
+   * @default Page ${page}
+   */
+  'aria-label'?: string;
+
+  /** Additional CSS class for the element. */
+  className?: string;
+
+  currentPage?: number;
+
+  /** The href for the anchor element. */
+  href: string;
+
+  page?: number;
+
+  /** Additional CSS styles for the element. */
+  style?: React.CSSProperties;
+
+  /** onClick function */
+  onClick: (event: React.UIEvent<HTMLElement>) => void;
+};
+
+const Page = React.forwardRef<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  PageProps
+>(({ page = 0, className, currentPage, ...props }, ref) => {
+  if (page < 1) {
+    return null;
+  }
+
+  const isCurrentPage = page === currentPage;
+
+  const ariaLabel = i18n._({
+    id: 'pagination.aria.page',
+    message: 'Page {currentPage}',
+    values: { currentPage },
+    comment:
+      'Default screenreader message for page button in the pagination component',
+  });
+
+  return (
+    <Button
+      pill
+      aria-label={ariaLabel}
+      {...props}
+      ref={ref}
+      rel="nofollow"
+      aria-current={isCurrentPage ? 'page' : undefined}
+      className={classNames(className, ccPagination.defaultPage, {
```

Comment:
I see that on hover the background color changes to `s-bg-primary-subtle-hover`, which looks like this:
<img width="207" alt="Screenshot 2024-01-08 at 12 31 09" src="https://github.com/warp-ds/react/assets/41303231/4167ab7f-3ed0-4ca8-96f2-97a9050f8a66">
Shouldn't the colors get darker on hover and active state like in the case of primary buttons? I couldn't see any design examples for pagination component so I thought I'd double-check here.

Reply:
The colors are from what @magnuh has in pagination.yml (https://github.com/warp-ds/css/pull/121/commits/dae034881766d9cf26247ce34dc3ff000a0dd20b)

Reply:
The colors are from the tokens sheet: https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1588196525&range=293:293
...so that is the correct behaviour. 🙂 
...but shouldn't the text for non active pages be link color (blue)? 🤔

Reply:
I now found a design [in Figma](https://www.figma.com/file/oHBCzDdJxHQ6fmFLYWUltf/Adi-sandbox%3A-Warp-variables-v2.0?type=design&node-id=818-35572&mode=design&t=LRe5aAQmwbCo9xYv-0), which also shows the primary-subtle-hover color.

Reply:
It also looks like in the design from Figma that the text should have a font-weight 300, while here it seems to have a font-weight of 700. 🙂

Reply:
Hmm, we don't support 300 font weight. Only 400 & 700

Reply:
Oh! 😄